rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -15,11 +16,23 @@ error_code _sys_game_board_storage_read(vm::ptr<u8> buffer, u8 code)
 		return CELL_EFAULT;
 	}
 
+	u64 psid[2] = { g_cfg.sys.console_psid_low, g_cfg.sys.console_psid_high };
+	u8 psid_bytes[16];
+	for (int i = 0; i < 16; i++)
+	{
+		psid_bytes[15 - i] = psid[i < 8 ? 0 : 1] & 0xFF;
```

Comment:
Simpler to use memcpy instead of this conversion.

Reply:
I think v128 is perfect for this as whole.

Reply:
Updated. Thanks a lot for the advice!